packages/aws-cdk/generate.sh

```diff
@@ -15,30 +15,20 @@ cat > build-info.json <<HERE
 }
 HERE
 
-# Build noctilucent package
-(
-  # Check out the submodule if it's not there already
-  if [ ! -f "vendor/noctilucent/Cargo.toml" ]; then
-    git -C ./vendor clone https://github.com/iph/noctilucent.git
-  fi
-
-  # update the package to the pinned commit hash
-  git -C vendor/noctilucent reset --hard HEAD
-  git -C vendor/noctilucent fetch && git -C vendor/noctilucent checkout 6da7c9fade55f8443bba7b8fdfcd4ebfe5208fb1
-
-  # Install wasm-pack if it's not there already
-  if ! command -v wasm-pack >/dev/null 2>/dev/null; then
-    echo "installing wasm-pack, this may take a while..."
-    cargo install wasm-pack
-  fi
-
-  pkgroot=$(cd $(dirname -- "$0") && pwd)
-
-  cd vendor/noctilucent
-  wasm-pack build --target nodejs                                               \
-    --out-dir="${pkgroot}/lib/vendor/noctilucent"                               \
-    --out-name=index                                                            
-
-  cd ../../lib/vendor/noctilucent
-  rm package.json
-)
+# Build noctilucent package in a Docker/Finch VM
+NOCTILUCENT_GIT="https://github.com/iph/noctilucent.git"
+NOCTILUCENT_COMMIT_ID="6da7c9fade55f8443bba7b8fdfcd4ebfe5208fb1"
```

Comment:
I agree with @rix0rrr this feels like we are reinventing versioning and releases.